politeiad/backend/gitbe/gitbe.go

```diff
@@ -2636,7 +2744,10 @@ func New(anp *chaincfg.Params, root string, dcrtimeHost string, gitPath string,
 		checkAnchor:     make(chan struct{}),
 		testAnchors:     make(map[string]bool),
 		plugins:         []backend.Plugin{getDecredPlugin(anp.Name != "mainnet")},
+		prefixCache:     make(map[string]struct{}),
 	}
+	g.popuplateTokenPrefixCache()
```

Comment:
Unhandled error.

Reply:
I'm no longer returning the error.

Reply:
Why not?  We want this to fail hard if there's an error populating the prefix cache.